sdk/cosmos/azure-cosmos-spark_3-1_2-12/dev/README.md

```diff
@@ -63,7 +63,7 @@ Use the partner release pipeline to release.
 For each release we need to go over the OSS compliance steps:
 
 1) Ensure the branch is tracked by Component Governance: https://dev.azure.com/azure-sdk/internal/_componentGovernance/106501?_a=components&typeId=6129920&alerts-view-option=active
-   master branch is automatically tracked. For the feature/cosmos/spark30 branch you need to manually trigger the run.
+   master branch is automatically tracked. For the master branch you need to manually trigger the run.
```

Comment:
We don't need the note anymore right? Because the Component Governance automatically tracks master